android/ReactAndroid/build.gradle

```diff
@@ -189,14 +189,14 @@ def findNdkBuildFullPath() {
         def ndkDir = property("ndk.path")
         return new File(ndkDir, getNdkBuildName()).getAbsolutePath()
     }
+
```

Comment:
Copied from RN 0.60+ 🙂, so I didn't bother making them in `react-native-lab` (since we plan on upgrading soon). Do you think it would make sense to make them even so?

Reply:
No, I just wanted to make sure we wouldn't accidentally overwrite these changes when syncing code into ReactAndroid.